main.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding-left: 2px;
```

Comment:
- [ ] Please remove this. As per the screenshot in this review, the padding to the left makes us miss the mark on the design of this page.